### DISCUSSION TITLE
For You section feedback

### DISCUSSION THREAD

**Discussion by Sanket-Mathur** [community/community#13262] [Other Feature Feedback, Questions, & Ideas] [answered]
I personally love this recently added feature, I no longer have my Following feed filled up with github-action cron-job alerts. 

However I didn't like the "Recommended based on people you follow" and "Recommended for you" in between updates from the users and organizations I've been following.

Addition Feature Request:
- There must be a "Not interested" button in order to silent some specific updates

**Comment by jfullstackdev** [comment, marked as answer]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
